content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -490,7 +491,7 @@ main()
 
 <a id="current"></a>
 
-The following behavior applies in 2.25.0 and earlier, and since 2.26.0 _without_ the `referentialActions` preview flag.
+The following behavior applies in 2.25.0 and earlier, and since 2.26.0 until 3.0.0 _without_ the `referentialActions` preview flag.
```

Comment:
```suggestion
The following behavior applies in 2.25.0 and earlier, and since 2.26.0 until 3.0.0 _without_ the `referentialActions` preview flag. That is, if the referential actions feature is **not** implemented.
```